integration/combination/test_api_settings.py

```diff
@@ -0,0 +1,176 @@
+from io import BytesIO
+from pathlib import Path
+
+import requests
+from parameterized import parameterized
+
+from integration.helpers.base_test import BaseTest
+
+from PIL import Image
+
+
+class TestApiSettings(BaseTest):
+    def test_method_settings(self):
+        self.create_and_verify_stack("combination/api_with_method_settings")
+
+        rest_api_id = self.get_physical_id_by_type("AWS::ApiGateway::RestApi")
+        apigw_client = self.client_provider.api_client
+        response = apigw_client.get_stage(restApiId=rest_api_id, stageName="Prod")
+
+        wildcard_path = "*/*"
+
+        method_settings = response["methodSettings"]
+        self.assertTrue(wildcard_path in method_settings, "MethodSettings for the wildcard path must be present")
+
+        wildcard_path_setting = method_settings[wildcard_path]
+        print(wildcard_path_setting)
+        self.assertTrue(wildcard_path_setting["metricsEnabled"], "Metrics must be enabled")
+        self.assertTrue(wildcard_path_setting["dataTraceEnabled"], "DataTrace must be enabled")
+        self.assertEqual(wildcard_path_setting["loggingLevel"], "INFO", "DataTrace must be enabled")
```

Comment:
Same error message as above

Reply:
the import error?

Reply:
The error message for the wrong logging level is `DataTrace must be enabled`, I think it should be something along `LoggingLevel must be INFO`

Reply:
will update it